CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,45 @@
+# Code of Conduct
+
+## What do we stand for?
+This Code of Conduct is designed to help all of us build a pleasant, productive, and accomplished team. Our Code of Conduct is not meant to burden the team with a bunch of needless rules, or to punish our valued employees for past behaviors. Rather, we are striving every day to make a hospitable environment for our great engineering teem, especially for those who have faced more adverse working environments in the past.
+
+We strive to build a culture of inclusivity, honesty, and integrity. As a team, we can ensure every employee feels heard and respected. Teamwork is a must. We’re all here together to make great products for our customers, and more importantly, a pleasant work environment for our colleagues.
+
+## Contribution
+It's important that everyone in the team are part of creating a learning environment.
+Every employee is valued equally. Teamwork is valued above all else. Understand that others might not have the same information you have, and be willing to help when asked.
+
+Being patient and helpful will boost trust among your colleagues and will help creating an environment where you and your colleagues won't feel insecure, embarrassed and afraid to ask questions when in need of help.
+
+It’s inevitable that we’ll all need help on occasion. By greeting each other with helpfulness and courtesy, we can have a productive workspace and a happy team.
+
+### Teach, don't tell
+If your colleague needs help understanding a task, take the time to walk them through it. 
+Show them how the task is performed so that they’ll be able to perform it next time. Have the mindset of teaching not telling.
+
+### Interjecting
+Interjecting, without being fully engaged, can be very disruptive. While you are encouraged to join conversations, take time to understand everything that’s being said so that you may assess the situation before providing help.
+
+## Time management
+Be respectful of others peoples time. When you need help, ask. But first, ask your colleague if they have time to help you with a problem. Be patient when waiting for responses.
+
+You’re responsible for keeping track of your work schedule. This means being on time to meetings, and providing advance notice if you’re unable to attend or will be arriving late.
+
+Keep things efficient by saving particular questions for meetings, or Slack conversations, so as to cut down on disrupting your colleagues during work hours. 
+
+## Giving and Receiving Feedback
```

Comment:
Minor, but either write: 

- "Giving and receiving feedback" 
- "Giving & receiving feedback"